comparison_interface/main/views/register.py

```diff
@@ -117,6 +118,13 @@ def _load_group_component(self, user_components: list):
         else:
             label_text = ''
             error_text = ''
+        if len(groups) > 6 and (len(groups) - 6) > 3:
+            group_length = int(len(groups) / 2) + 1
+            groups = [groups[:group_length], groups[group_length:]]
+            group_columns = True
+        else:
+            groups = groups
+            group_columns = False
```

Comment:
Is this easier to read? I have suggested '>= 10' as an equivalent operation, used '\\' instead of integer cast and I think 'groups = groups' is redundant.
```suggestion
        if len(groups) >= 10:
            group_length = (len(groups) + 1) // 2
            groups = [groups[:group_length], groups[group_length:]]
            group_columns = True
        else:
            group_columns = False
```